docs/blog/2019-04-03-how-we-think-about-product-at-gatsby/index.md

```diff
@@ -0,0 +1,130 @@
+
+The goal of the product org is to understand the pain points of our customers, create relevant personas, and do the necessary research, discovery, and prototyping so that we can provide solutions they find valuable.
+
+## What does the product organization do?
+
+Since we are putting together a group focused on product, we also want to have outcomes that keep us on track. **An outcome measures the level of performance or achievement that occurred based on the thing we created and measured.** That means that we can have outcomes based around company goals, around products, or even around specific features.
+
+### Identifying the outcomes we want for persona and business problems
+
+We focus all of our conversations around problems at Gatsby. We want to discover, identify, and map out problems that exist both for the business and for our customers and community. By identifying how impactful and prevalent the problems are, we can start deciding if and when we want to tackle solving them.
+
+### Driving alignment with company Objectives and Key Results (OKRs)
+
+Identifying and writing down these problems also helps us write our objectives and key results because the objectives are created to list **what we want to do** based on the problem we are trying to solve, and the key results show **how we will measure that it’s been done**.
+
+Each team in Gatsby has objectives and key results that they are responsible for hitting.
+
+The product organization is responsible for making sure our key results are set and that any dependencies on other teams are clear, so they can successfully achieve their key results as well.
+
+### Creating an outcome-based roadmap that gives the company a sense of where we are with product development or features
+
+As we learn more about our community and our customers through research, discovery, and metrics, we will create a roadmap that shows what outcomes we want to provide for people. This is different from a traditional roadmap because we won’t be focusing on features, but rather on the problems we’ve seen and the outcome that would show we’ve solved the problem.
+
+### Giving internal stakeholders high integrity commitments
+
+There is always some measure of conflict between sales, marketing, engineering, and product. Each group has a goal that it’s trying to accomplish. These goals should create a checks and balances system that keeps teams researching, prototyping, building, and communicating with customers. As an example, engineering wants to build the best possible product from a technical standpoint. Product wants to create value for the users. Sales wants to sell as much of the product as possible to the broadest audience on the shortest timeline. Marketing wants exciting improvements that generate buzz and create opportunities for publicity.
+
+All of these goals are important, but create a necessary tension that forces us to make collaborate and make decisions in order to keep moving forward.
```

Comment:
"and" instead of "but" makes sense to me here

Reply:
Added, thank you!